Armadillo/src/main/java/com/scribd/armadillo/playback/ExoPlaybackExceptionExt.kt

```diff
@@ -47,4 +67,18 @@ internal fun ExoPlaybackException.toArmadilloException(): ArmadilloException {
     } else {
         UnexpectedException(cause = this, actionThatFailedMessage = "Exoplayer error")
     }
+}
+
+private fun DataSpec.toAnalyticsMap(context: Context): Map<String, String> {
+    return mapOf(
+        "uri" to uri.toString(),
```

Comment:
In general it is ok to have .toString() but we won't be able to differentiate between value being null vs empty string right? We can bubble that up as well.

Reply:
This uri property is always non null -- also toString() returns "null" for null objects and "" for empty string